tsconfig.json

```diff
@@ -55,6 +55,8 @@
     "skipLibCheck": false,
     "module": "commonjs",
     "lib": [
+      "es2019",
+      "es2018",
```

Comment:
can you please explain what these additional libs bring to the transpiled es6 target build?

Reply:
Object.fromEntries pardi ! :)

Reply:
`Object.fromEntries()` is available in the current version of Electron used in Thorium (`8.2.1`), both in Node (v`12.13.0`) and in Chromium's V8 (v`8.0.426.30`).
See:
https://node.green/#ES2019-features
and:
https://github.com/mdn/browser-compat-data/blob/036d00b5853b8711d94a2b163dfba0fcafba06e2/javascript/builtins/Object.json#L703-L754

Reply:
I know

but without "es2019" :

`[2] ERROR in [at-loader] ./src/utils/deepClone.ts:14:31 
[2]     TS2339: Property 'fromEntries' does not exist on type 'ObjectConstructor'.
[2] 
[2] ERROR in [at-loader] ./src/main/lpfConverter.ts:153:53 
[2]     TS2339: Property 'fromEntries' does not exist on type 'ObjectConstructor'.`

Reply:
Ah yes, because the TypeScript compiler transpiles to ES6 in the current configuration, some constructs are modified in the output JS (I guess `Object.fromEntries` becomes `ObjectConstructor` something ...)

Reply:
indeed